notsup Valid OS:    darwin,linuxnpm

Hi!

is there a chance to get this adapter running under Windows 10?

br
Holger

Hi aivus,

I'm using Windows 10 and when I try to install the Adapter in iobroker I get this fault:

---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 

$ ./iobroker url "https://github.com/inwaar/gree-hvac-client" --debug
install https://github.com/inwaar/gree-hvac-client/tarball/master
npm install https://github.com/inwaar/gree-hvac-client/tarball/master --production --save --prefix "C:/Program Files/iobroker/SmartHome" (System call)
npm
 ERR! code EBADPLATFORM
npm
 ERR! notsup Unsupported platform for gree-hvac-client@1.2.4: wanted {"os":"darwin,linux","arch":"any"} (current: {"os":"win32","arch":"x64"})npm
 ERR!
 
notsup Valid OS:    darwin,linux
npm 
ERR! 
notsup Valid Arch:  any
npm 
ERR! 
notsup Actual OS:   win32
npm 
ERR! 
notsup Actual Arch: x64

npm ERR! A complete log of this run can be found in:npm ERR!     C:\Program Files\iobroker\SmartHome\env\npm-cache\_logs\2019-10-23T17_46_20_389Z-debug.log

process exited with code 0

---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 8< ---- 

So it looks like the adapter can't be installed on a Windows OS...

best regards
Holger

--- a/package.json
+++ b/package.json
@@ -16,10 +16,6 @@
   "engines": {
     "node": ">=8.11.0"
   },
-  "os": [
-    "darwin",
-    "linux"
-  ],
   "devDependencies": {
     "jsdoc-to-markdown": "^5.0.0",
     "mocha": "^6.2.0"

